quiz.rb

```diff
@@ -7,7 +7,13 @@
 # it should return false if age is less than minimum_age
 
 def is_of_age? age, minimum_age
-  # your code goes here
+  puts age
+  puts minimum_age
+    if age.to_i <= minimum_age.to_i then
```

Comment:
you dont need the `if..else` statement here. `age.to_i <= minimum_age.to_i` will evaluate to true or false. also, as you probably noticed, this is backwards